storage/pkg/command/gateway.go

```diff
@@ -222,6 +228,118 @@ func rules(cfg *config.Config, logger log.Logger) map[string]map[string]interfac
 	}
 }
 
+func mimetypes(cfg *config.Config, logger log.Logger) (m map[string]map[string]string) {
```

Comment:
just a style note, I wouldn't mix named returns with not-naked returns, it feels redundant to me. For style I'd prefer not named returns and explicit returns, even if that means 2 extra lines of variable declarations

Reply:
ok, removed the named returns